package.json

```diff
@@ -15,9 +15,9 @@
     "test": "jest",
     "lint": "eslint .",
     "format": "eslint . --fix",
-    "babel": "babel --no-babelrc --plugins=@babel/transform-block-scoping,@babel/transform-modules-commonjs,@babel/transform-react-jsx,@babel/proposal-class-properties,@babel/proposal-object-rest-spread,@babel/transform-flow-strip-types src --copy-files --out-dir dist --ignore '**/__tests__/**','*.ts'",
+    "babel": "babel --extensions '.js,.ts,.tsx' --no-babelrc --config-file=./babel.config.publish.js src --ignore '**/__tests__/**'  --copy-files --source-maps --delete-dir-on-start",
     "tsc": "tsc",
-    "build": "yarn babel --out-dir dist && yarn tsc",
+    "build": "yarn babel --out-dir dist && del 'dist/**/__tests__' && yarn tsc --emitDeclarationOnly",
```

Comment:
Added `del 'dist/**/__tests__'` because when using `--copy-files`, babel cli will copy files specified in `--ignore` and there seem to be no way to disable this behavior. I was lurking in their issue tracker in the past and they suggested to use a more featured build tool 🤷‍♂️